pydra/engine/task.py

```diff
@@ -158,11 +158,11 @@ def run(self, cache_locations=None, cache_dir=None, **kwargs):
         # TODO add signal handler for processes killed after lock acquisition
         with FileLock(lockfile):
             # Let only one equivalent process run
-
+            #dj: for now not using cache
```

Comment:
why? if you don't provide a cache_location it will always run in a temp directory. is there something that happens when you enable cache?

Reply:
i think it was not working, but it was at the time that many things were not working, so will try to uncomment it